src/main/java/com/minecolonies/coremod/client/render/RenderChiefBarbarianMob.java

```diff
@@ -0,0 +1,51 @@
+package com.minecolonies.coremod.client.render;
+
+import com.minecolonies.coremod.entity.ai.mobs.EntityChiefBarbarian;
+import net.minecraft.client.model.ModelBiped;
+import net.minecraft.client.renderer.entity.Render;
+import net.minecraft.client.renderer.entity.RenderLiving;
+import net.minecraft.client.renderer.entity.RenderManager;
+import net.minecraft.client.renderer.entity.layers.LayerBipedArmor;
+import net.minecraft.client.renderer.entity.layers.LayerHeldItem;
+import net.minecraft.util.ResourceLocation;
+import net.minecraftforge.fml.client.registry.IRenderFactory;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Created by Asher on 5/6/17.
+ */
+public class RenderChiefBarbarianMob extends RenderLiving<EntityChiefBarbarian>
+{
+    /* default */ private final ResourceLocation mobTexture = new ResourceLocation("minecolonies:textures/entity/chiefbarbarian1.png");
+
+    /* default */ private static final float SHADOW_SIZE = 0.5F;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "SHADOW_SIZE" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1068)